compiler/rustc_parse/src/parser/attr.rs

```diff
@@ -58,9 +56,10 @@ impl<'a> Parser<'a> {
                     let span = self.token.span;
                     let mut err = self.sess.span_diagnostic.struct_span_err_with_code(
                         span,
-                        fluent::parse_inner_doc_comment_not_permitted,
```

Comment:
How do we handle this?
maybe we need a new macro to mark the content need to be translated.